test/Libraries/WorkflowTests/PackageValidationTest.cs

```diff
@@ -132,17 +132,17 @@ public void CreateMeshFromSolids()
             AssertNoDummyNodes();
 
             // check all the nodes and connectors are loaded
-            Assert.AreEqual(13, CurrentDynamoModel.CurrentWorkspace.Nodes.Count());
-            Assert.AreEqual(14, CurrentDynamoModel.CurrentWorkspace.Connectors.Count());
+            Assert.AreEqual(17, CurrentDynamoModel.CurrentWorkspace.Nodes.Count());
```

Comment:
@ke-yu any idea what changed in master to change the number of nodes in the graph when this file is opened? Some migrations?